Jakefile

```diff
@@ -161,17 +161,30 @@ function generateChromeManifest() {
   });
 }
 
+/**
+ * Exports the files to a target, used to export into mozilla-central
+ *
+ */
+function exportFiles(target) {
+  let mcLocation = getMozillaCentralLocation();
+  let extTargetDir = path.join(mcLocation, target);
+  jake.rmRf(extTargetDir);
+  jake.cpR(BUILD_DIR, extTargetDir);
+
+  console.log(`Exported built sources into ${extTargetDir}`);
+}
+
 desc(`Builds the extension into the ${BUILD_DIR}/ directory`);
 task("build", ["building:cleanup", "building:copy"], () => {});
 
 desc("Exports the sources into mozilla-central");
 task("export-mc", ["build"], () => {
-  let mcLocation = getMozillaCentralLocation();
-  let extTargetDir = path.join(mcLocation, "browser/extensions/webcompat");
-  jake.rmRf(extTargetDir);
-  jake.cpR(BUILD_DIR, extTargetDir);
+  exportFiles("browser/extensions/webcompat");
+});
 
-  console.log(`Exported built sources into ${extTargetDir}`);
+desc("Exports the sources into the mozilla-central for android");
+task("export-mc-android", ["build"], () => {
```

Comment:
Input from others if we want to put 1 source code in m-c:
1. Use gecko-dev/toolkit
2. Dev-tool team plans to move to system addon, they may have the problem as us.